### ISSUE TITLE
Initial Story Development

### ISSUE THREAD

**Issue by datahead8888** [Secretchronicles/TSC#68]
Forum Discussion:
- http://www.secretmaryo.org/phpBB3/viewtopic.php?f=1&t=9958
- http://www.secretmaryo.org/phpBB3/viewtopic.php?f=10&t=3594

Tasks:
- Develop rough, initial storyline ideas
- Develop some possible world ideas based on initial storyline ideas
- Develop some possible power up ideas based on this storyline
- Develop some possible boss ideas based on this storyline
- Post ideas to forum and engage the community in discussion
- Modify ideas based on community feedback as needed
- Write initial draft of main story document
- Modify story document based on community feedback

Additional tasks for artwork, etc, will evolve out of this task.

**Comment by datahead8888** [comment]
This is where I've been playing with new ideas for a while: http://wiki.secretchronicles.de/Story%20Development%20Document%20-%20Version%202

It's still a WIP version, though, so it's not "ready for review yet"

**Comment by Quintus** [comment]
[IRC discussion](http://chatlogs.secretchronicles.de/htmllogs/2015-01-01.log.html#msg-2015-01-01T23:48:01+00:00)

**Comment by datahead8888** [comment]
@tristanheaven @Luiji @brianvanderburg2 @carstene1ns @default0 @DarkAceZ @dlpenguinlover @Bugsbane @kirbyfan64 @xet7 @giby @Quintus @sydneyjd @sauer2 @SonOfBowser @Aakburns  @burlito  -

A new story draft is ready for review by the team at:
http://wiki.secretchronicles.de/Story%20Development%20Document%20-%20Version%202

This draft does not include the suggestions to include a female heroine character or other playable characters as discussed at http://forum.secretchronicles.de/topics/209.  More discussion is still needed there.

Please review and provide feedback - the story is intended to have a large influence on a number of decisions.

**Comment by Bugsbane** [comment]
Generally, it sounds great. A couple of notes:

1. Vo.do is a very cool website that distributes creative commons films. I'd hate to have their name (Vodo) used as one of the evil, arch villains to be destroyed when Vodo (the website) are actually so Copyleft friendly...

2. Faythe - sounds like the word Faith, which makes it sound like the whole game is a piece of religious propaganda. Rather than make the game seem either pro or anti any religion, I suggest we pick names that don't evoke any particular religion at all.

3. Final level is on the Evil arch villains ship with weapons. Does this not sound a little too similar to SMB 1 (the last level is on Bowser's floating ship with canons all over it).

4. I have trouble visualizing urban skyscraper levels fitting into what I imagine the game to be. Can anyone provide links to any other different games using the kind of look that's envisioned for these levels? Maybe if I can see the kind of thing we're talking about I might understand better. Currently it sounds like it would stick out a lot from the other levels.

5. I like the idea of the Fox like Faythians. That sounds like the kind of character I'd like to design. Doe we have any links to visual inspiration material for these?

6. "the story is intended to have a large influence on a number of decisions". Can I just confirm that if people have ideas for other levels or ideas over time that there will be room to look at incorporating these (where there is general consensus), and that the story (which is more than just a story as it goes into specific level designs) is seen as a general goal, rather than a final, definitive, completed list of what everyone will be working on? If new members feel that they aren't allowed to introduce new ideas, it would really lower the motivation to contribute.

7. I like how the story ties together a range of locations from the jungle, to the library, to the factories etc.

Overall it looks like a fun story, with some memorable characters and a good setup to explore a full world. I recall reading in an earlier story draft something about Alex realizing Dr M wasn't actually evil at the end, and that it was largely himself jumping to conclusions. I quite liked that angle and am wondering if that reveal would still be at the end of this?

**Comment by datahead8888** [comment]
> Final level is on the Evil arch villains ship with weapons.

No, we've been discussing making the final world the Great Library.  The ship would be an intermediate location.  I could edit the order in the document if that is making this unclear.

> Can I just confirm that if people have ideas for other levels or ideas over time that there will be room to look at incorporating these

Yes, of course - where there is consensus and where we can make the ideas work.  I just meant that if we want the story to work in the game, elements of the game have to draw on it rather than be largely random.  I've had a little trouble getting story feedback in the past.  Also, the story will always be a work in progress to at least some extent as new ideas keep coming in.

I won't comment on the other suggestions until others have had a chance to reply.

**Comment by Bugsbane** [comment]
Ok. I don't think it's an issue if the ship isn't the last level, like in SMB.

> if we want the story to work in the game, elements of the game have to draw on it rather than be largely random

Couldn't agree more with you on this statement. The thing that's always driven me crazy with SMC has been the piecemeal, inconsistent design born out of many contributors with no source material to guide them. It gets old having to explain over and over that having assets look like nothing else in the game is a problem. That extends to the story, the music, the art etc. Certainly I welcome anything that can help unify everyone's visions, as long as everyone gets a say in it's creation.

In a similar way, I'd really like to have debates around the look of the game revolve more around what should go into the visual style guide, which all artists would then try and follow, than whether the latest picture posted as a new asset draft looks "cool" or not. In that sense, I think we're trying to accomplish the same thing, me with the visuals and you with the story.

**Comment by datahead8888** [comment]
> In a similar way, I'd really like to have debates around the look of the game revolve more around what should go into the visual style guide, which all artists would then try and follow, than whether the latest picture posted as a new asset draft looks "cool" or not. In that sense, I think we're trying to accomplish the same thing, me with the visuals and you with the story.

Yes, the two discussions are very related; this actually is why I tried to finish the story draft now -- we need to think about what we all want to do.  I have some thoughts on the style guide that I can post later.  I've thought on the suggestions you gave before as to how to give more concrete suggestions (within the bounds of my own art experience).  Once we get those posted, we can see what works well for the game as a whole and let the team make decisions on it.

**Comment by Bugsbane** [comment]
Sounds good.

**Comment by Quintus** [comment]
The story draft is very neat. Thank you for your work, @datahead8888! Some notes:

> Alex stepped into the opening, turning on his flashlight.

We could actually give Alex a flashlight that can be activated. When we implement darkened levels, this could be very cool. The battery could run out over time, so the player has to carefully decide when to use the flashlight and when better not to.

> They are the final, missing piece of his collection. With them he will have the power to rule this world." 

We will have to substantiate this a bit. If I tell someone you need the magic SD card to rule the world, nobody would believe me probably. Over time, the player should find out _why_ these chronicles allow ruling of the world.

> Faythe - The home country of the Faythian fox people.

As for these, I agree with @Bugsbane’s critisism. We would probably create unintentional side interpretations.

As for order: The Great Library was meant to be the last level. This was probably a bit mixed up in the draft.

>  have trouble visualizing urban skyscraper

We have a skyscrapers background in the game already and have a nice city music posted on the forums, so it was natural to have something with cities. I think it was also talked about that topic on IRC earlier, but I’d have to search my chatlogs for that.

> Can I just confirm that if people have ideas for other levels or ideas over time that there will be room to look at incorporating these

The story is not final yet, and even when we have decided on it, it will take several releases before it will actually be “implemented” properly. It will probably see several refinements. And even when all is done, there will be room for new ideas. We have campaigns in the game; we can always try to continue with new ideas in another campaign. As an example, we could shift in time and tell stories that happened before or after the main story we are developing now. Who knows what Alex’ children are going to do, or what Dr. M’s parents were doing? Or we could shift in place and tell the story of characters that only play a minor role in the main story. There are many possibilities, and I don’t want the game to stop when the story we are working on now has been finished. I’ve been watching FreeDink for some time, and they are prisoners of their own story because they didn’t do that. They are adding in material in the middle of the game, but when you have played an earlier version, you always know where you’re going to. (Disclaimer: That was the situation about two years ago. I have not played the game since then). I find such an attitude dangerous, because it overly stretches a story that had a good length before. You should know when to stop. All good stories come to an end, but there may be many side stories we still can tell. Or, to quote a famous German author, “that’s another story that shall be told another time” (the author is Michael Ende; the phrase is used at several places in “The Neverending Story” to hint at lose ends where one could dig up an entire book of its own).

> I recall reading in an earlier story draft something about Alex realizing Dr M wasn't actually evil at the end, and that it was largely himself jumping to conclusions

Absolutely. I’d like that too. He didn’t really notice that he was doing things that were considered evil by others. All he wanted to do is the protect the world with the knowledge he gathered... Or something like this. I leave it up to datahead’s imagination :-)

Valete,
Quintus

**Comment by datahead8888** [comment]
This is only one possibility for the story.  Feel free to challenge any part of it, much as Bugsbane suggested not using city levels.  If I disagree, I'll state my point of view, but it's good to discuss these ideas.  I'm not attached at all to the ruins or monster outpost worlds if we want to pull something out, though these worlds may work well.

Quintus said:
> We will have to substantiate this a bit. If I tell someone you need the magic SD card to rule the world, nobody would believe me probably. Over time, the player should find out why these chronicles allow ruling of the world.

Yes, we will.  We also should decide where the Secret Chronicles came from and for what purpose they were created.  This sort of thing tends to get ironed out in further drafts.

Quintus said:
> As for order: The Great Library was meant to be the last level. This was probably a bit mixed up in the draft.

The order of locations was not intended to be chronological, but we should probably make it chronological to prevent confusion.  I will move the Great Library item in this draft to prevent further confusion.

Quintus said:
> We have a skyscrapers background in the game already and have a nice city music posted on the forums, so it was natural to have something with cities. I think it was also talked about that topic on IRC earlier, but I’d have to search my chatlogs for that.

...and Bugsbane said:
> Currently it sounds like it would stick out a lot from the other levels.

Yes, we have both a city background and a song that might work as a metro song.  The idea of city levels came out of the background graphics.  If given to the fox people, cities would imply these people have technological sophistication, allowing them to fight Dr. M.  I'd be interested in not going with the "primitive oppressed animal people" paradigm and instead have them be formidable.

Also, since we have been seriously looking at having technologically advanced factory levels (and maybe a ship level), the city levels will not unusually stand out.  The expanse is intended to be a series of worlds full of wilderness, which will let us have classic ice, snow, desert, forest, and plains levels.

Bugsbane said:
> Vo.do is a very cool website that distributes creative commons films. I'd hate to have their name (Vodo) used as one of the evil, arch villains to be destroyed when Vodo (the website) are actually so Copyleft friendly...

The name Vodo comes from "DanielVodo".  This is the user name of a spam bot we banned and tried to delete all posts for on secretmaryo.org.  The server ran out of memory, and as a result, residual traces of Daniel Vodo's posts show up on the main boards.  It's annoying, but there's nothing we can do.  Thus Quintus was joking that there could be a persistent enemy with the same name in the game.

Bugsbane said:
> Faythe - sounds like the word Faith, which makes it sound like the whole game is a piece of religious propaganda.

I just came up with something here.  I agree we don't want overt references to religion in the game, since it will quickly start polarizing our audience.  TSC takes no position on faiths, so this is not needed.

We could do another hack on this word -- Fayeth, Fayenth, etc, just as long as it doesn't sound too much like faith.

Bugsbane said:
> I like the idea of the Fox like Faythians. That sounds like the kind of character I'd like to design. Doe we have any links to visual inspiration material for these?

I came up with the description off the top of my head, but possible inspirations are Star Fox (http://www.classifiedgamerz.com/miyamoto-discusses-new-pikmin-star-fox/)  and the Chronicles of Narnia.  The slightly curled ears are not directly borrowed.

> I like how the story ties together a range of locations from the jungle, to the library, to the factories etc.

Thank you.  I wanted it to be relatively easy both to use elements from Secret Maryo Chronicles as we see fit and add things as the team suggests them.

Bugsbane said:
> I recall reading in an earlier story draft something about Alex realizing Dr M wasn't actually evil at the end, and that it was largely himself jumping to conclusions. I quite liked that angle and am wondering if that reveal would still be at the end of this?

Yes, this is what was in the original draft.  Dark Ace Z suggested making Dr. M "all evil" (see: http://www.secretmaryo.org/phpBB3/viewtopic.php?f=1&t=9958) , so I tried exploring that in this draft.  I agree it could be more interesting to have a villain who's not as evil as you might think.

Here are a few possible angles on this:
* Dr. M took the actions he did because he believed he was stopping a greater evil.  This is kind of like the villain in the movie "Watchmen".  This villain commits murders in the belief that these actions will ultimately allow him to stop nuclear war between nations.  He hates what he does but does it because he sees no alternative.  We of course would not have outright murders in the game, but the evil for the sake of good concept is interesting.
    * Kirby's Adventure also explores this theme when Kirby fights King Dedede, only to discover King Dedede took the actions he did in order to stop a greater enemy.
* Alex brings the fight to Dr. M in order to help the fox people, only to discover they were manipulating him.  He thought it was Dr. M who is evil when in reality it is the fox people who are evil.
* Neither the fox people nor Dr. M is really "evil".  They are just opposing sides in an unfortunate war, much like many civil wars in history.

Quntus said:
> You should know when to stop.

I agree we can't always add an idea to an existing story, but I do like expansive or long stories :)

Another reason I wanted to finish this draft early was because the character back story (age of the character and his profession) are directly related to the art design in task #264.  We will want to discuss decisions on these elements.

This story also has yet to integrate the suggestions about a heroine (or other playable game characters) at: http://forum.secretchronicles.de/topics/209.  I think we also need females on the enemy's side.  I'd like to leave Dr. M as a male, though.

**Comment by Bugsbane** [comment]
I'd just like to state, for the record, that regardless of the history behind the name, that I explicitly oppose using the name of a friendly, progressive, copyleft, pay what you want website as the name of one of the most evil villains in the game. The people behind Vodo are good people, helping to bring more copyleft, DRM free content to the world. To me this is similar to calling Malus something like "Linux".


> Neither the fox people nor Dr. M is really "evil". They are just opposing sides in an unfortunate war, much like many civil wars in history.

This would be my favorite option, although I like all the ones you wrote there.

>  I think we also need females on the enemy's side. I'd like to leave Dr. M as a male, though.

Agreed. As long as the as the main "good" character is male, I'd rather have the main "bad" one of the same gender to avoid one gender being "evil" and the other "good". I would like to see female characters in the game though, although I strongly suggest we start with a "good" female character before introducing an "evil" one, to avoid any misconception that one gender could be being demonized. If we really want to get all inclusive, I'd suggest a character or two who aren't entirely cast as male or female with any of the stereotypical cues (ie dresses, moustache, long eyelashes etc). In a fantasy kind of setting, this is usually easiest to do just by having a few "genderless" (by apearence) characters.

**Comment by Bugsbane** [comment]
Hey, if we do have a playable female character, she could be called "Alice" to match "Alex" heh heh. Actually, I'm thinking how in the recent indie game Gianna Sisters, you could swap between two characters with different powers at any time. It could be cool to have Alex turn into Alice and vice versa at various points, either defined by the level, or activated by the player. I agree with datahead8888 though, that this is best left for after we have the basics nailed down.

**Comment by datahead8888** [comment]
> I'd just like to state, for the record, that regardless of the history behind the name, that I explicitly oppose using the name of a friendly, progressive, copyleft, pay what you want website as the name of one of the most evil villains in the game. The people behind Vodo are good people, helping to bring more copyleft, DRM free content to the world. To me this is similar to calling Malus something like "Linux".

How well known are they?  There are many popular web sites, magazines, companies, organizations, etc.  There is always a chance that any villain name will stomp on someone.  Also, if we make it clear it's "Daniel Vodo", it's could be viewed as a different kind of Vodo.  I respect your view on this but think we should be cautious in making blast lists of names.

> This would be my favorite option, although I like all the ones you wrote there.

I was leaning towards the "good for the sake of evil" one, but it depends entirely on how we develop any one of these options.

> Agreed. As long as the as the main "good" character is male, I'd rather have the main "bad" one of the same gender to avoid one gender being "evil" and the other "good".

We could also have a major male and major female villain, each with opposing goals.  Final Fantasy VII had two different enemies: Sephiroth and Shinra.  It's just another possible idea.

For the genderless characters, some of the fox people are a good candidate :)

> you could swap between two characters with different powers at any time

There was some discussion about having more than 2 playable characters (perhaps many characters), each with different abilities (http://forum.secretchronicles.de/topics/209 - toward the bottom).  Just imagine how fun it would be to unlock the option to "Play as Dr. M" :)  I don't know about you, but I would love that...

**Comment by Bugsbane** [comment]
> There is always a chance that any villain name will stomp on someone

This is true, however in this case we would be doing it knowingly, in advance, with something that would be dirt simple to change at this point. The name also isn't just similar. We're using their name exactly and knowingly, and there's just no need. I would also suggest that because of what Vodo do with DRM free, copyleft media, that their audience are likely to have large overlap with TSC as an open source game.

What I don't see, is any intrinsic value to keeping the name Vodo that makes it worth demonizing people who are trying to basically the same thing as us in a different field.

What about if we just changed / added a single letter to make it something like Vodox, Zodo, Kodon etc.? A quick search on even these doesn't turn up anything as blatantly problematic as Vodo, and in 10 minutes I'm sure we could come up with a list of many other options.

> I was leaning towards the "good for the sake of evil" one

I like that one, too.

> We could also have a major male and major female villain, each with opposing goals

We could, although it seems like more complexity in the story than we need.

> For the genderless characters, some of the fox people are a good candidate 

Absolutely.

> I would love that

That would be very cool!

**Comment by sauer2** [comment]
For the protocol: After an intense brainstorming session Bugsbane and I came to the conclusion that

* Alex is fine
* Dr. Malus is fine
* and Daniel Vodo would be better off as Ázdhôr (Vodox | Zodo | Kodon)

**Comment by Quintus** [comment]
> just imagine how fun it would be to unlock the option to "Play as Dr. M" 

All your magic are belong to us.

Seriously, we can do that at some late point in the game, when the player defeats Dr. M. and finds out he actually isn’t the evil guy he thought. They then unite to... whatever. I don’t know. Maybe the game just ends there.

> Daniel Vodo

Good god, I wouldn’t have thought that our spambot friend would manage to be evil even after his death! Arrr...

I’m fine with not using that name. I just thought it was a neat reference to our problems and an appropriate way to express our disgust about spamming. But hey, it was just an idea. I don’t want to discredit any copyleft groups.

> Dr. Malus is fine

Maybe we can make a bit of a pun here. “mălus” (with short a) means “evil” in Latin, and “mālus” (with long a) means “apple” in Latin (→ this is the exact coincidence that created the poisonous apple tree in the Genesis book -- it was a translation error I heard. The fruit of evil was translated as an apple that way). We could have him a vegetarian or so as a side note.

Valete,
Quintus

**Comment by datahead8888** [comment]
> Ázdhôr 

We could use this as his name and drop Daniel as suggested.  Many native English speakers will probably view it as a foreign name, which is good.

Maybe we could make a plot reference to the spam bot situation by having Ázdhôr be in charge of army of awful, PER5SISTENT robots (like the spam bots) :) Either he creates them or is a robot himself...

> Seriously, we can do that at some late point in the game, when the player defeats Dr. M. and finds out he actually isn’t the evil guy he thought. They then unite to... whatever. I don’t know. Maybe the game just ends there.

I was thinking it might be a bonus unlockable where you can play through the game again as Dr. M.  In this case it does not have to integrate with the plot (though having Dr. M join you is interesting -- this might inspire someone to create a new game called TSC RPG :+1: )

**Comment by Danfun64** [comment]
If there is going to be both a hero and a heroine, it would be nice to have a 2 player coop (nsmbw-esque) and competitive (smb3/smw-esque), and even race (smbd gbc) modes.

**Comment by Quintus** [comment]
I have expressed my preference towards a single-player game earlier in the forums as I really see this game as a single-player one. However, there are other TSC people like datahead and Luiji who also have interested in multiplayer mode, so it may happen nevertheless. Adding any kind of multiplayer to this game is basically a rewrite of the entire codebase, because it is very, very centered around a single player. If any multiplayer is added, it will have to be restricted to levels that are made specifically for it; ordinary levels will most likely break under multiplayer mode. Anyway, multiplayer discussion should be taken out in ticket #76.

Vale,
Quintus

**Comment by Luiji** [comment]
@Quintus My current position on multiplayer is that it isn't practical. We'd have to create a completely new level set around it for it to work well, and although we may one day choose to do that, it's really out of bounds to our current resources (at least by my estimates). Many other tasks need to be completed before we get into this.

In the case of multiplayer, we can choose to make the plot completely independent of it, since we can have the multiplayer campaign use another story. In my opinion, plot is campaign-specific (or at least specific to a set of campaigns, which could be considered a series).

**Comment by datahead8888** [comment]
My view is that we should never scratch multiplayer off our list, but I agree with Luiji that we will have to focus on other things for a quite a while.  De-Marioization, story development, mechanics development, and some of the game architecture/API decisions are far more pressing at the moment.  Some of the game architecture problems that prevent multiplayer mode should be cleaned up regardless; they may turn out to limit us on other features.

We've already talked about having multiple playable characters (2 or more).  This could provide an interesting feature in single player alone, especially if each player has different abilities.  If we move to multiplayer later, these characters will provide an easy backdrop for multiplayer mode.

If and when we implement multiplayer mode, we should probably first focus on multiplayer on the same computer, since it is significantly easier to implement.  I would be interested in eventually having network multiplayer if other programmers become interested in programming it.  Given the sheer amount of architecture required for network multiplayer, however, that feature must be heavily discussed before being implemented.  As for multiplayer level sets, this could be introduced in a new campaign eventually.

It sounds like Luiji and I are in agreement on multiplayer :) Multiplayer is something that keeps coming up again and again (both for TSC and in the old SMC forums) - it's definitely worth talking about.  

@Luiji - I have never heard any of your thoughts on any of the story drafts.  What are your thoughts on the current draft (http://wiki.secretchronicles.de/Story%20Development%20Document%20-%20Version%202)?  I can't take your feedback into account without knowing your thoughts :)

**Comment by Danfun64** [comment]
@datahead8888 
I know you weren't talking to me, but I have some ideas on taking ideas from both drafts, as well as how to handle the multiple character issue, but I haven't written them up yet. All I will say for now is that my plan is mostly based off version 2, but has Alex as a kid using the current Maryo graphics (vectorized and without the "m" ' s of course).

I'll let you know when and if I finished my idea of the plot.

**Comment by datahead8888** [comment]
> I know you weren't talking to me, but I have some ideas on taking ideas from both drafts, as well as how to handle the multiple character issue, but I haven't written them up yet. All I will say for now is that my plan is mostly based off version 2, but has Alex as a kid using the current Maryo graphics (vectorized and without the "m" ' s of course).

@Danfun64 - You might want to attend our IRC meeting if you are able to, since we were planning on discussing both the age of Alex and how to change his graphics.  These are major story and art decisions.  At the very least, it might make sense to pursue a new look for him with it being a new fork.  Related topics are how cute the game should be and how similar TSC should be to SMC.  See: http://team.secretchronicles.de/~quintus/general-discussions/1st-general-discussion.txt

**Comment by Danfun64** [comment]
On second thought, forget my idea. I think it might be too disturbing for a family friendly game.

**Comment by datahead8888** [comment]
> On second thought, forget my idea. I think it might be too disturbing for a family friendly game.

No, please share it - it's good to discuss ideas.  We always try to have adequate discussion before actually changing anything.  I just meant to say that the age of the hero, the art style of Alex, and SMC similarities are all things we were discussing already.

**Comment by Danfun64** [comment]
I'll try to gather the details before the meeting will occur.

**Comment by Danfun64** [comment]
I need help with naming Faythian characters. Do you have any suggestions on how they should be named? Should it be something exotic, or should it be relatively similar to human names?

**Comment by datahead8888** [comment]
Something different than ordinary human names (or names most people are familiar with) would probably be more interesting, but it still needs to be names that people can still pronounce and make sense of.

We probably are not going to call them Faythians, though, since concerns were raised that this sounds too close to "faith".  I was thinking about doing a different hack on this word to change some syllables around, etc.  You might see what names you can come up with for them.

**Comment by Danfun64** [comment]
I think I ran out of time for writing my unofficial story development document. I have to give time for the developers to read the thing, so I am attaching it to this post. Keep in mind that it is incomplete and if decide to go with my ideas, you should wait until I can produce a second version of the document which is more complete. Something I didn't mention. The art style for the game would remain the same for the most part, especially in the earlier worlds, but gradually get more disturbing in a way that doesn't clash, especially in World 6,7, and 8. The art style for standalone/animated cut scenes (if they are implemented) would be anime-esque.

http://www.mediafire.com/view/igovgwcf4pizc8b/TSC-DF64.odt

**Comment by datahead8888** [comment]
> I think I ran out of time for writing my unofficial story development document

We may not get much time to read a document this close to the meeting, so don't worry about it.  Just be ready to explain things in IRC as needed.  Also, we don't have to absolutely decide on everything tomorrow.  We can follow up on decisions later if needed.  It will probably take some time for everyone to think on new ideas.

**Comment by Danfun64** [comment]
I decided to release the next version of my document. It is still incomplete, but not nearly as much as it was before. Yes, the story leads straight to the next campaign, and like Super Mario Bros: The Lost Levels, the difficulty will be more challenging. As dark as the situation in even the best ending of the first campaign in my document is, I wanted to show there is hope. In a way, the ending is similar to the ending of the movie version of Fellowship of the Ring. People have died, and the main villain is only getting more powerful, but there is hope. Maybe a fools hope, but a hope nonetheless.

http://www.mediafire.com/view/5nrraupwh19s1e7/TSC-DFv1.odt

**Comment by datahead8888** [comment]
@Danfun64 - It may be a while before I can post a review of your document.  I've got a ton of stuff going on in school and need to catch up on some of this stuff.  You also have a good length, which will take longer to review.  Thank you for offering your ideas for the story.

**Comment by Quintus** [comment]
@datahead8888 If you find the time, it’d be great if you can take a look on the document. Here are my thoughts so far:

The story suggested is very intense, and actually coming towards a kind of positive ending is difficult. What I do like is the idea to have multiple different endings in the game, which should be doable with modifications of the overworld and campaign code. To generalise the approach, I think we should add scripting to at least the campaigns so campaign authors can define endings themselves without having us to change C++ code -- the levelset and thus the story should not be tied to the underlying C++ source.

There’s a lot of dying going on near the end, and in all cases poor Dr. M. is not going to be alive afterwards. I don’t think it is possible to merge this with our previous ideas regarding the story. It is a rather dark story, which is an interesting idea for a platformer but in my opinion does not fit the goal for a game that can be played by children and provides additional value to elder people.

As for the pipes, please keep in mind we try to reduce their use because the pipes are a clear Mario resemblance. There’s ticket #281 that talks about replacing them with teleporters or similar things. There were also shrines suggested, and logs, so that the pipes would be one transport mechanism among others, depending on the surrounding ambience.

danfun, your story draft describes several possible endings and it’s easy to lose overview. Would you mind using Graphviz or a similar tool to create a diagram that shows the possible pathes? That’d make it easier to discuss specific points in one of the story pathes. If you use Graphviz, please also post the Graphviz sourcefile so it can be easily modified.

My opinion is only one of many. I’d like to hear opinions by @Luiji and @DarkAceZ, but especially by @datahead8888 as he is the author of the original story draft which I see you have taken up at some parts.

Valete,
Quintus

**Comment by Danfun64** [comment]
I told you it would end up being too dark :P

**Comment by Danfun64** [comment]
I admit that the story got a little *too* dark. I will lighten the story a little in my next version of the document, toning down the death.
Also, the two fox parents wouldn't have come *completely* out of nowhere. They would have been mentioned in a cutscene.

**Comment by Danfun64** [comment]
http://www.mediafire.com/view/3hi6nr2e9l38432/TSC-DFv2.odt

The faythians don't know much about the dark lord, whom I still haven't named yet. That's why in some of the endings, they welcome his taking over from Dr. M with open arms. Yes, Dr. M lives in some of the bad endings, and it is implied that he lives in the good endings. I also spared the lives of Haakon and his parents, and explained why he is sour and why he isn't playable in the second campaign.

I know that I have been uploading updated versions of my story on a weekly basic, but I doubt that I will be able to post V3 next week because I will be vacationing. Tell me if you think my newest changes made the story lighter or darker.

**Comment by Luiji** [comment]
@Danfun64 I've yet to read the whole document (or any of the various story documents, really...) but I'll say it piques my interest, at least. It seems much deeper than I would've expected for this game.

**Comment by datahead8888** [comment]
@Danfun64 - I apologize for the long wait for my feedback.  I had started typing a response for your first document, but then you posted a second document.  I have been slowed down a lot with school recently.  It's probably best I just go ahead and post what I had written for *document version 1*, and hopefully I can see what you changed some time soon in document version 2.  My response is long, but your document was long and was also based on the document I had posted.

My #1 suggestion is not to rush the story.  I really appreciate that you are trying to get all the gaps filled in, @danfun64, but good stories tend to take a lot of time.  I intentionally did not try to fill in all gaps in my document because I knew we needed to discuss some of these things.  My last document did make some fairly major proposals, but I knew this would need to be discussed at length.

Age of Alex - As I said, I think it might be good to consider making him an adult.  If he is set as a child, it may make people think the game is only targeted at children.  As for the discussion about Kingdom Hearts, a teenager is not a child.  A teenager could be a possible compromise here, though.

Fox characters: I like that you've already made some proposals for fox characters, since we had not developed any fox characters yet.  I like the fox twins concept, though I'm not sure green would work for a color.  Having an older fox is an interesting idea, though I'm personally a little hesitant to play on the grandpa concept too much.

Helen: We really need to discuss the female character's name at length with the team.  It took a while to agree on Alex.  We will probably want to open a ticket on this and get talking.  The escape artist angle might be interesting to talk about more; if we use this, we could adapt her in game abilities to this.

For Dr. M, you wrote:

> His true intentions are indecipherable, and his views of 
right and wrong are unintelligible. It is very difficult to tell if he is truly evil, or if he is a 
paranoid man who wants to be left alone.

This seems a little “too” vague for me.  I like Bugsbane's suggestion not to have him be purely evil, but we need to be more specific than this.

The idea I've been throwing around with some people is to have Dr. M be a type of dictator who really believes that he is helping the people he rules.  He thus forces his ideologies on people, thinking he will improve their way of life.  This also gives an opportunity to explore some social issues, such as what licensing Dr. M makes the people use (ie. the Dr. M General License).

Gage – We've been needing new subvillains.  A clone is one possibility.  If we use this character, I would propose more than just changing his clothing and hair.  Maybe we could also change his eyes or face to make it look a bit more sinister.  It also might be interesting to have a “clone lab” in the monster factory, with clones of Alex bobbing up and down in tubes.  It's just a thought.

I can't really say I like the parasite concept or the idea of entering Gaith's body later in the game.  For having a world in Gaith's body, jumping on red blood cells as platforms is an interesting idea, but I just have trouble seeing this in TSC.  For the parasite concept, this is discussed a little more further below.  It also might be interesting to have him say more than “Engage the enemy.”  Otherwise, he should probably not speak at all.

Berry blocks – Blocks in the game can hold more than just berries.  The name probably needs to be more specific.  I do agree that we need to explain how these blocks were created.

Opening of the story – having natives chase Alex in the normal world is an interesting idea.  This might develop his character while we transition to the entrance to the other world.  We will need to be careful, though, not to be too similar to Indiana Jones.

Talking about the origins of the gems is a good idea.  We should explain where all the gems in the game came from.  We also might want to explain how cookies are randomly placed in the different worlds, since they will not grow naturally as fruits do.

> “Step-daughter? But that means...you're a girl!?!” Admittedly, it wasn't easy to tell she was a girl at 
first, having clothing that resembled Alex's wear. However, the voice should have been a dead 
giveaway, as is her long, braided hair. Helen rolls her eyes, “What? You honestly never saw a girl 
before?” She asked.

I agree that having a female playable character in the game is a good idea, but this suggests a rivalry between males and females.  I think it's best to avoid this in the game.  I would rather just assume they are equals and never imply there is a problem.

> Basically, a Ph.D. is a piece of written writing that states that you passed a certain standard 
of intelligence.

Overall a PhD shows that a person has passed a standard for research ability.

> “The Malevolent Chronicles document the quest of a man to learn great knowledge. At least, that's how it begins. Halfway through them, he succeeds in his quest, and in the process becomes an 
unrecognizable force of evil. The rest of the Chronicles, what makes them truly Malevolent, are the 
dark spells that it contains. Some of the pages give instructions on how to awaken several great evils. 
Other pages list rare animal species that could be used for evil purposes, and where to find them.
“Legend states that after the book was completed, it developed a horrible mind of its own. Nobody is 
able to prove that though, since it has disappeared from the face of the earth thousands of years ago...or
so we thought.

The idea I last had was to have each book in the Secret Chronicles be a different attribute: ie. Malevolence, Benevolence, Curiosity, etc, much like the elements of fire, water, air, and earth.  Together, they hold the ability to open a gateway to another world known as the Magic Realm, which holds unspeakable power.  This realm might also have a greater evil that could be explored in later campaigns, perhaps having a new villain, such as a “darker version” of Dr. M.  This would give an excuse to have Dr. M join the team as a playable character.

Neither idea is wrong.  It's just a question of what direction we want to go.

> the good doctor

This is a great tagline :)

Worlds - I had not meant to start planning all of the worlds in my document – I originally wanted to cite story related locations.  Some of these may becomes worlds, but it depends on later discussions.  I think it may be best to move a little slowly on this.  I don't think we should plan exact levels yet, either.

I had figured the expanse might become multiple worlds – this would let us have any number of worlds with elements such as forests, deserts, ice, etc.  It gives us an easy way to use any number of levels that don't directly fit into the plot and allows us to have transitions in the types of levels the user sees (Fox people worlds → Expanse → Dr. M related worlds).  Your Borderlands and River of Sorrow might help address this concern, but we still should not constrain ourselves as to what the Expanse contains.  We may have additional types of worlds as we continue to talk, however.  

The River of Sorrow sounds interesting.  Having river levels is something the old Mario games usually did not have.  It is a nice alternative to just having underwater and above the surface water levels (though these would still be fun to have).

Quintus said:
> There’s a lot of dying going on near the end, and in all cases poor Dr. M. is not going to be alive afterwards. I don’t think it is possible to merge this with our previous ideas regarding the story. It is a rather dark story, which is an interesting idea for a platformer but in my opinion does not fit the goal for a game that can be played by children and provides additional value to elder people.

I was leaning toward having a story that appeals toward children with fun, fast moving somewhat cute graphics and teenagers/adults at the same time with a deep story and references to more advanced concepts.  If we include violence of any level, though, we will completely lose the child audience and have a game only playable by teenagers and adults.  Even if we add more campaigns with different plots, we will still have this problem.

I like the idea of having some dark elements, but things such as death or serious cruelty should probably be glossed over.  We could easily replace killed characters with captured characters in Dr M's layer, for example, which also has the benefit of being able to play those characters again later.

I really can't see the slugs/spiders/zombies concept working.  Anything involving characters having parasites damaging their brains, etc, may fall into the above category somewhat.  I honestly have trouble seeing it in the type of game we've been discussing, too.

Quintus said:
> As for the pipes, please keep in mind we try to reduce their use because the pipes are a clear Mario resemblance. There’s ticket #281 that talks about replacing them with teleporters or similar things. There were also shrines suggested, and logs, so that the pipes would be one transport mechanism among others, depending on the surrounding ambience.

Yes, we can't have heavy references to pipes in the game.  Pipes will probably only be used in building structures where you would expect to see them.  There are a lot of other objects we can use in place of pipes, but we still will not want it to seem like a direct swap out from the Mario games for the plot.

Also, we cannot have a “warp zone”.  That is a direct Mario reference.  We could talk about having some sort of other transportation system that allows you to quickly traverse the larger world, however.

On another note, we need to decide if everyone is fine with city levels or not.  Bugsbane originally objected and seemed to rescind a little bit when we explained where the idea came from.  Does anyone else have concerns with city levels?  They would be something different than the Mario games, but it's a question of what types of levels and themes people enjoy.

**Comment by Danfun64** [comment]
Then it is agreed that the ideas I came up with are too dark, and would only work if I was aiming for a E10+ or T rating, which I doubt *you guys* are.

Feel free to cannibalize whatever aspects of my document you want. I need to step away from contributing to the story for a while.

**Comment by datahead8888** [comment]
> Feel free to cannibalize whatever aspects of my document you want. I need to step away from contributing to the story for a while.

I would not call it "cannibalizing".  We had said in IRC we would likely accept/reject individual elements of the document and would not necessarily just use the entire document as-is.  However, feel free to offer as many ideas or documents as seem reasonable - it really helps us.  We just have to move slowly on finalizing ideas; good fiction takes time.  I posted the exact same thing for a song recently in the music forum.

Being the author of the document from which you derived, though, I'm the most likely person to give you point by point feedback.  This actually is the kind of feedback that authors need in order to evaluate their work; you would get the same detail of feedback if you sent your draft to a professional.

Thank you for sharing your ideas - it's really appreciated :)

**Comment by Danfun64** [comment]
I know that I already decided against doing much more with my story, but I wanted to point out a couple more things about it, both involving the clone Gage. First, he was never meant to be evil, it was only the circumstances of his creation that made him act on the side of evil at first. Second, the aforementioned creation.

The Dark Lord, having managed to cause Alex to bleed a little, carefully scoops up the blood and brings it back to his lair. He pulls out an old spellbook, and find the page on creating beings from their blood. He recites an incantation on the blood, and it begins to transform into a baby made out of a clay like substance. He carves through the head area of the clay baby, revealing another clay-like lump inside, resembling a brain. He puts a Mesmo Slug on top of the brain area, and closes the the incision. After he smooths out the head so as to hide the incision, he looks at the spell book again, and recites a spell that causes the baby like figure to grow into a clay boy, closely resembling Alex. The Dark Lord turns the pages a couple times until he gets to the part that says "hardening". He they recites that spell. The clay child starts looking more human, its body oozing out a chemical that causes it to harden the boy to glass. The Dark Lord breaks the grass, leaving a human child that looks just like Alex, with no life in it. Finally, he shocks the child with electricity, causing it to wake up. The boy's eyes are completely blank...

This information, combined with the rest of my material, might be too dark for a kids story, but there you go.

**Comment by sauer2** [comment]
> This information, combined with the rest of my material, might be too 
> dark for a kids story, but there you go.
Still, I like it. TSC doesn't need to be designed for children. A lot of 
good things have been ruined by being adapted for children. On the other 
hand, a lot of good things not designed for children - with an evil 
twist - run on childrens networks...

**Comment by datahead8888** [comment]
@sauer2 / @Luiji - if you have not done so, please read the original story documents.  I have no doubt we should make uses of ideas from Danfun64's document, but you will not have perspective on what we're discussing without reading the older documents.  I've been heavily advocating not having a child's story for a while.

**Comment by sauer2** [comment]
@datahead8888: It's the second proposal in the wiki, right?
Maybe I will get to this tomorrow.

Am 12.04.2015 um 23:58 schrieb datahead8888:

**Comment by datahead8888** [comment]
Yes, the last draft I had posted was http://wiki.secretchronicles.de/Story%20Development%20Document%20-%20Version%202.  It sounds like we all have more reading to do before we discuss anything more :)  Even Quintus mentioned he had more reading to do (before deciding the heroine's name).

**Comment by sydneyjd** [comment]
@Danfun64 SOunds like a interesting story, but im not sure if i quite agree with going towards spells and other stuff so much. It sounds very African, and this game is definately not that. ;)
I think we may be able to use it in a way though.

@datahead8888 Your version fits the whole TSC theme very well, even though a couple words are a little too long. :stuck_out_tongue:  

Maybe change ""So it's true . . ." he said." to "So it is true..."he thought.

Just my opinions. There seems to be quite a bit of debate. :wink:

**Comment by datahead8888** [comment]
I was busy with grad school, then other things such as a personal project and the SFML port, but I should have unassigned myself from this if I didn't have time.  I apologize for the delays on this.

I will try to have a review of the prior story submissions in a week, give or take.

**Comment by datahead8888** [comment]
I went back and read through all of Danfun64's second story revision.  My thoughts are pretty much the same -- I think the slug and zombie concepts are a bit too violent for the game, given that children will be part of our audience.  They would make this into a horror game.  I do, however, like the fox characters and a couple of the locations such as the River of Sorrow.  Please see my prior post (a little bit up) for an in depth review of version 1 of the document.

A couple new things I noticed - I like the idea of having Lord Azdhor overthrow Dr. M and becoming the greater threat.  I also like the idea of having multiple endings, if we can spare the time to make them.

@Danfun64 - your document was very, very long.  Most people just skimmed it.  Please break such things into multiple types of documents.  We can try to borrow ideas from your document if you are still fine with it.  You're also welcome to offer more suggestions and propose new documents, but you will have to reach agreement with the team.

I'm going to call for an IRC team meeting soon, but I would like to compose a LaTeX document relating to story principles I've been thinking about for that meeting.  It may take some time to finish.  If we urgently need the meeting soon, I may have to skimp on the formatting/images I hoped to include in it.

**Comment by datahead8888** [comment]
I have now posted the story principles document, outlining major decisions to be made and offering suggestions:
http://team.secretchronicles.de/~datahead/DataheadStoryPrinciples.pdf

@Bugsbane - this document discusses decisions related to the art style -- we really will need your feedback when we hold our IRC meeting talking about this.
@Danfun64 - we will formally talk about your submission in the next meeting.
You had said:

> Feel free to cannibalize whatever aspects of my document you want. I need to step away from contributing to the story for a while.

...thus we have your permission to use things from it, but we still really want your input on this.

We are looking at holding an IRC meeting, in which we will both discuss story decisions and other items.  See this link for topics being developed:
http://forum.secretchronicles.de/topics/387

**Comment by Bugsbane** [comment]
Ok, when is the irc meeting?

**Comment by datahead8888** [comment]
> Ok, when is the irc meeting?

We're still ironing that out.  We're just about to enter the phase where everyone can post their thoughts on topics (in document form if long).  After that we'd ideally allow some time for everyone to read those documents before the meeting.

**Comment by datahead8888** [comment]
I was showing story documents to someone when I realized that Quintus's story proposal is not linked to here.  Here it is:
https://github.com/Secretchronicles/documents/raw/master/gd/documents/pdf/16-004.pdf

It is likely we will spend more time on the story discussion again after some of the current work on stabilizing devel is done, for the SFML and Cegui ports.  Feel free to offer thoughts in the mean time, but keep in mind I was intending to post more thoughts and/or a draft myself.  Quintus is correct we need to get the story draft done with some time semi-soon, however.

**Comment by datahead8888** [comment]
Some thoughts on Quintus's draft:
* Akan Albin - I honestly would probably go with a different name than Akan.  A female, Greek name might be a possibility, since Alexander is a Greek male name.  Albin could work as a last name, however.
* Dr. M as an old man - I always imagined him as a middle aged man, though this is just my view.  I especially wouldn't go with having him be a desperate old man who barely realizes what is happening around him when he is betrayed by Lord Azdhor -- this makes me feel like he is too weak.
* Medieval / early industrial setting - I was thinking it might be interesting to have some parts of the game with a science fiction component, especially the monster factory and Dr. M's ship.  A possible compromise here would be to have different regions of the game have different types of technology and/or cultures.  Not everyone in our fictional world has access to the same technology.  In this way we could blend many different story/art elements into the game eventually.
* Use of the motif 12 - This is a very good use of literature, though I'm honestly a bit hesitant to put limits on the number of worlds or levels.  Granted, 12 worlds is a lot (and some of the original Mario games only had 8).  Still, though, I think it would be interesting for this campaign to have as many as 20 worlds eventually, after some years.
* I like the idea of using press articles to advance the story.
* I liked how you incorporated the idea of having Lord Azdhor be a greater enemy than Dr. M.
* I like Lord Azdhor's cloud tower
* I'm a bit unsure about Dr. M dying once the Secret Chronicles are destroyed.  It is an interesting idea but also a dark ending for the game.

**Comment by Quintus** [comment]
On Thu, Jan 19, 2017 at 09:21:41PM -0800, Chris Jacobsen wrote:
> Some thoughts on Quintus's draft:
> * Akan Albin - I honestly would probably go with a different name than
> Akan.  A female, Greek name might be a possibility, since Alexander is
> a Greek male name.  Albin could work as a last name, however.

The naming is not fixed. I just came up with what was on the top of my
mind. A female greek name would work well, as it is consistent. Nice
idea, I'll have to look some up.

It didn't become clear from my draft, but I want to suggest that once
the heroine becomes available, the player can choose which hero he uses
to play a level. Alex and the heroine should have slightly different
capabilities so that they can reach different places or even have to
solve a level differently. Once Dr. M. becomes available, one can reach
previously unreachable places due to his advanced magic facilitities by
choosing him for playing an already resolved level.

I'm not sure whether one should allow change of the characters
mid-level, as it might complicate level design. I'd not be against that,
though, as it is logical, because otherwise the group would be left
behind.

> * Dr. M as an old man - I always imagined him as a middle aged man,
> though this is just my view.

I'd not be against this. I chose that appearance as I wanted to resemble
the classic figure of the "old wise man", and then parody it with that
old wise man being betrayed. Is there a Shakespeare reference for that? :-P

> I especially wouldn't go with having him
> be a desperate old man who barely realizes what is happening around
> him when he is betrayed by Lord Azdhor -- this makes me feel like he
> is too weak.

Okay, but to allow for the betrayal, you need to either weaken Dr. M. as
I have done, or strengthen the Lord's cleverness. "Barely realizing" is
not the term I would use for Dr. M.; he's not *that* old and
uncapable. All I wanted to achieve is his credulity (I hope I got that
word right: someone who believes too easy what others say to him), and I
based this on the "evil aura" the possesment of a number of Chronicles
books causes.

> * Medieval / early industrial setting - I was thinking it might be
> interesting to have some parts of the game with a science fiction
> component, especially the monster factory and Dr. M's ship.

We are going to need some science fiction anyway in these settings, and
I'm absolutely fine with that. We are creating a fantasy world after
all, and as long as it doesn't contain contradictions in it, we can do
whatever we want.

> A possible compromise here would be to have different regions of the
> game have different types of technology and/or cultures.  Not everyone
> in our fictional world has access to the same technology.

I like that.

> * Use of the motif 12 - This is a very good use of literature, though
> I'm honestly a bit hesitant to put limits on the number of worlds or
> levels.

While it would probably be ideal, I wouldn't enforce a limit of 12
levels on each world. I do however want the Great Library world to have
exactly 12 levels for symbolism.

> Granted, 12 worlds is a lot (and some of the original Mario
> games only had 8).

I think 12 worlds for the main campaign is doable, though not in the
upcoming 2.1.0 release, which I want to point out is not required
offhand. There is no need to have everything complete right in the next
release. We can (and will have to) evolve over time.

> Still, though, I think it would be interesting for
> this campaign to have as many as 20 worlds eventually, after some
> years.

Hmm. I think we should postpone that question until we have enough level
material for that. We might think different by then, though I would
argue that at this point, we should start adding a second campaign.

> * I'm a bit unsure about Dr. M dying once the Secret Chronicles are
> destroyed.  It is an interesting idea but also a dark ending for the
> game.

I wanted to give the game a grave touch towards the end, something to
think about. Not as brutal as it was outlined in other submissions, but
still something that leaves a more permanent impression. I would even
like to further refine this point so that at the end the game conveys
some kind of message. If we can do this without Dr. M. dying, I'd be
fine with that.

Vale,
Quintus

-- 
Blog: http://www.guelkerdev.de
PGP/GPG ID: F1D8799FBCC8BC4F

**Comment by kirbyfan64** [comment]
I would just be a bit cautious about the whole press thing. They did something like that in Final Fantasy XV (you learn of plot information by reading the newspaper and/or listening to the radio), and to say people were whining is an understatement. I like the idea, but it should probably be really obvious.

**Comment by Skarfester** [comment]
Some thoughts:

- I think the female character should be eligible from the begining, for all campaigns and worlds (choosing in the options the default one when playing levels alone). I read somewhere the name Helena. I like it.

- Alex being a thief? I prefer both Alex and Akan-Helena-wathever being treasure hunters, some sort of Indiana Jones. That gives them a motif for exploring worlds, and opens the path for making new campaigns in the search of other treasures.

- I like the idea of Dr. M being a wise man corrupted by the Chronicles. That gives a way to "recover" him and help the heroes at the end.

- Azdhor-Vodo (Dr. M right's hand). A character wich is true evil who betrays dr. M and becomes the bad guy? Absolutely!

- I don't see Dr. M dead and the books burned at the end. Maybe for paying his bad actions he becomes a sort of eternal guardian of the chronicles...

- I like the idea of different worlds with different technology levels.

**Comment by datahead8888** [comment]
I haven't done any serious work on this task for a while; I would rather focus my main effort on writing my own book as I get time.
I had some more ideas for the TSC story; I'll try to post them in a bulleted list later.
It will be up to the team what they want to do with them; otherwise Quintus actually posted a good sized draft already.

If nothing else gets done with the story for a while, I might dink with it over time, but nothing is guaranteed.  Thus I am unassigning myself from this.

**Comment by Quintus** [comment]
As nobody seems to have any objects, I finally close this very long ticket now. The base story is then set. Any further story refinement should receive discussion elsewhere, preferably on the mailing list as it's not a one-off TODO item probably.
